chrome/content/zotero/xpcom/data/item.js

```diff
@@ -4181,6 +4181,7 @@ Zotero.Item.prototype.toJSON = Zotero.Promise.coroutine(function* (options) {
 	
 	obj.dateAdded = Zotero.Date.sqlToISO8601(this.dateAdded);
 	obj.dateModified = Zotero.Date.sqlToISO8601(this.dateModified);
+	if(obj.accessDate) obj.accessDate = Zotero.Date.dateToISO(Zotero.Date.sqlToDate(obj.accessDate));
```

Comment:
So this converts to UTC, whereas `Zotero.Date.sqlToISO8601` does not. What exactly do we want here?